bundles/GraphicServiceOrder/Resources/views/createOrderForm.html.twig

```diff
@@ -12,13 +12,14 @@
     </div>
     <h4 class="mt-4">{{ 'service_order_form.profile.header'|trans }}</h4>
     <div>
-      <div class="mb-2">
-        <span class="label"><strong>{{ 'service_order_form.profile.name'|trans }}:</strong></span>
-        <span class="value">Martin Yde Granath</span>
+      <div class="form-group no-gutters">
+        <label>{{ 'service_order_form.profile.email'|trans }}:</label>
+        <span class="value">{{ user_email }}</span>
       </div>
-      <div class="mb-2">
-        <span class="label"><strong>{{ 'service_order_form.profile.email'|trans }}:</strong></span>
-        <span class="value">my@aarhus.dk</span>
+      <div class="form-group">
+        <label>{{ form_label(form.full_name) }}:</label>
```

Comment:
Maybe this could be handled by using `form_row` (https://symfony.com/doc/4.1/reference/forms/twig_reference.html#form-row-view-variables)?

Reply:
indeed, fixed